### PR TITLE
Integration tests not running on pipeline

### PR DESCRIPTION
**Pull request by Dannyps**
# Readiness checklist

- [ ] I added/updated unit tests.
- [ ] I added/updated integration tests.
- [x] I ensured that the PR title is good enough for the changelog.
- [x] I labeled the PR.

### CODE REVIEW

**Comment by Dannyps** [comment]
We were skipping the building of the tests, which resulted in them not being ran.

File: .ci/integration-test.ts [hunk 1]
```diff
@@ -3,5 +3,5 @@
 import { $ } from "zx";
 
 await $`docker compose -f ./docker-compose/docker-compose.test.yml up test-consumer-api test-admin-ui test-seed-mssql test-seed-client -d`;
-await $`dotnet test --no-restore --no-build "Backbone.sln" --filter "Category=Integration&TestCategory!~ignore"`;
-await $`docker compose -f ./docker-compose/docker-compose.test.yml down`;
\ No newline at end of file
+await $`dotnet test "Backbone.sln" --filter "Category=Integration&TestCategory!~ignore"`;
```
Comment (tnotheis) [inline .ci/integration-test.ts:6]: Run separate `dotnet restore` and `dotnet build` commands. This will make the output easier to read.

**Review by tnotheis — CHANGES_REQUESTED** [review]

**Review by tnotheis — APPROVED** [review]

**Comment by tnotheis** [comment]
I removed the `bug` label because only bugs in the actual products (Consumer API, Admin UI, ...) should be labeled as such.

**Comment by tnotheis** [comment]
@Dannyps Don't you want to merge this PR?

**Comment by Dannyps** [comment]
Nah, it's good like this.

Kidding, I forgot 🤣 I was convinced I had set it to automerge.

**Pull request merged**